From 1477e7907521322bc0b7ac2fa315124066b75d5a Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sat, 7 Sep 2019 09:32:21 -0700
Subject: [PATCH] dgit-maint-{merge,debrebase}(7): Use git fetch --tags
 upstream

We recommend using a remote called 'upstream' and so we can recommend
a command which will be faster and less likely to error out.

Suggested-by: Ian Jackson <ijackson@chiark.greenend.org.uk>
Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 2 +-
 dgit-maint-merge.7.pod     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 27c97aa3..4a0b5a52 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -353,7 +353,7 @@ release, and importing that release using git-debrebase(1).
 
 =over 4
 
-    % git fetch --all --tags
+    % git fetch --tags upstream
 
 =back
 
diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 37a02613..19950d4e 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -362,7 +362,7 @@ to git), you can just run dpkg-buildpackage(1) or debuild(1) instead.
 
 =over 4
 
-    % git fetch --all --tags
+    % git fetch --tags upstream
 
 =back
 
-- 
2.20.1

